backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/ManifestManager.java

```diff
@@ -198,6 +206,17 @@ Manifest getManifest(final BackupIdentifier id) {
     }
   }
 
+  public Collection<Manifest> listManifests(final BackupIdentifierWildcard wildcard) {
+
+    return blobContainerClient
+        .listBlobs(new ListBlobsOptions().setPrefix("manifests/"), null)
```

Comment:
:wrench: Have you considered doing the same that the GCS store does? It tries to use as much of the wildcard as possible to construct a longer prefix. This makes the iteration more efficient in most cases. See `wildcardPrefix` in the GCS manifest manager.

Reply:
Good point! 🙌  This class and the filesetManager share similar logic and are good contenders to use some shared code across the backups.